common/shared.c

```diff
@@ -251,3 +251,90 @@ void show_perf_mr(int tsize, int iters, struct timespec *start,
 	printf("usec/xfer: %f", ((float)elapsed / iters / xfers_per_iter));
 	printf(" }\n");
 }
+
+void ft_csusage(char *name, char *desc)
+{
+	fprintf(stderr, "Usage:\n");
+	fprintf(stderr, "  %s [OPTIONS]\t\tstart ping server\n", name);
+	fprintf(stderr, "  %s [OPTIONS] <host>\tpong given host\n", name);
```

Comment:
The ping usually comes before the pong.  :)  That said, I don't like the description including the term ping/pong anyway.  'start server' and 'connect to server' seem sufficient.